tests/src/data/use-cases/pet/db-appoint-specie.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { type LoadSpecieByNameRepository } from '@/data/protocols'
+import { DbAppointSpecie } from '@/data/use-cases'
+import { type AppointSpecie } from '@/domain/use-cases'
+import { makeFakeSpecieRepository } from '@/tests/utils'
+
+interface SutTypes {
+  sut: DbAppointSpecie
+  specieRepositoryStub: LoadSpecieByNameRepository
+}
+
+const makeSut = (): SutTypes => {
+  const specieRepositoryStub = makeFakeSpecieRepository()
+  const dependencies: AppointSpecie.Dependencies = {
+    specieRepository: specieRepositoryStub
+  }
+  const sut = new DbAppointSpecie(dependencies)
+
+  return {
+    sut,
+    specieRepositoryStub
+  }
+}
+
+describe('DbAppointOtherSpecie Use Case', () => {
+  const specieName: AppointSpecie.Params = 'any_name'
+
+  describe('LoadSpecieByName', () => {
+    it('Should return specie equal other and specieAlias equal specieName when specieName is not equal to specie in db', async () => {
+      const { sut, specieRepositoryStub } = makeSut()
+      const otherSpecie = {
+        id: 'any_id',
+        name: 'Outros'
+      }
+      jest.spyOn(specieRepositoryStub, 'loadByName')
+        .mockResolvedValueOnce(undefined)
+        .mockResolvedValueOnce(otherSpecie)
+
+      const result = await sut.appoint(specieName)
+
+      expect(result).toEqual({
+        specie: otherSpecie,
+        specieAlias: specieName
+      })
+    })
+
+    it('Should return specie and specieAlias equal Outros when specieName is equal Outros', async () => {
```

Comment:
This test is doing the same thing as the previous one

Reply:
Same reason:
https://github.com/PetJournal/petjournal.api/pull/42#discussion_r1353708466

Reply:
actually, this test case is when the user choose the option "other" and write in the field "outro", this this test prevents a future bug.